lib/common/network/api/characters_api.dart

```diff
@@ -10,18 +11,42 @@ part 'characters_api.g.dart';
 abstract class CharactersApi {
   @factoryMethod
   factory CharactersApi(Dio dio) =>
-      FakeApi.characters.isEnabled
-        ? FakeCharactersApi()
-        : _CharactersApi(dio);
+      FakeApi.characters.isEnabled ? FakeCharactersApi() : _CharactersApi(dio);
 
   @GET('')
   Future<Characters> getCharacters();
+
+  @POST('')
+  Future<Characters> addCharacter();
 }
 
 class FakeCharactersApi implements CharactersApi {
   @override
-  Future<Characters> getCharacters() {
-    // TODO: implement ping
-    throw UnimplementedError();
+  Future<Characters> getCharacters() async {
+    // Simulate fetching characters from a fake API
+    final fakeCharacters = [
+      const Character(
+        title: 'Fake Character 1',
+        imageUrl: 'https://via.placeholder.com/600/92c952',
+        description: 'This is a fake character description.',
+      ),
+      const Character(
+        title: 'Fake Character 2',
+        imageUrl: 'https://via.placeholder.com/600/7286a7',
+        description: 'Another fake character description.',
+      ),
+    ];
+
+    final fakeResponse = Characters(characters: fakeCharacters);
+
+    await Future.delayed(const Duration(seconds: 2));
+
+    return fakeResponse;
+  }
+
+  @override
+  Future<Characters> addCharacter() async {
```

Comment:
why do we return Future<Character> here, but here https://github.com/arrrtem22/character_viewer/pull/11/files#diff-7b7f72fa6f5b258c8531378615b98142f95fd2a88ea3bfd9b67d2d33064d7c46R14 we return nothing